Usecase 3/delay-plugin/target/classes/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,20 @@
+Manifest-Version: 1.0

```

Comment:
why are class files and manifest files checked in? Please create gitignore file

Reply:
Forgot to add the target directory to the .gitignore file. Apologies. This has been fixed now